protos/bottle/fulfillment/v1/line_group.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineGroup {
+  string id = 1;
+  string name = 2;
+  int32 quantity = 3;
+
+  bottle.catalog.v1.Product catalog_product = 4;
+
+  enum LineGroupCategory {
+    LINE_GROUP_CATEGORY_UNSPECIFIED = 0;
+    LINE_GROUP_CATEGORY_DESKTOP = 1;
+    LINE_GROUP_CATEGORY_LAPTOP = 2;
+    LINE_GROUP_CATEGORY_SERVER = 3;
+    LINE_GROUP_CATEGORY_APPAREL = 4;
+  }
+  LineGroupCategory category = 5;
+
+  bool rush = 6;
+  double rush_price = 7;
+
+  enum LineGroupWarrantyType {
+    LINE_GROUP_WARRANTY_TYPE_UNSPECIFIED = 0;
+    LINE_GROUP_WARRANTY_TYPE_STANDARD = 1;
+    LINE_GROUP_WARRANTY_TYPE_EXTENDED = 2;
+  }
+  LineGroupWarrantyType warranty_type = 8;
+  double warranty_price = 9;
```

Comment:
Originally this was just a string but they have different tax codes so enum would be better

Reply:
Carl mentioned moving Warranty into the catalog so it'd just be a regular line on the order.

Reply:
Realized a potential problem with that. Unlike a display that is a separate product, warranty is tied to a specific product. It’s really warranty for serial number X.

Reply:
Warranty is technically in the catalog but as an option. This seems to be the ideal place for it because it can be a different price per model. How we apply it technically per product (as an option or something special) is up for debate though